docs/Using-Fleet/Adding-hosts.md

```diff
@@ -206,4 +206,51 @@ installation should appear as the same host in the Fleet UI. If other settings a
 entries will appear in the Fleet UI. The older entries can be automatically cleaned up with the host
 expiration functionality configured in the application settings (UI or fleetctl).
 
+## Grant full disk access to osquery on macOS
```

Comment:
@GuillaumeRoss Should there be two sections here, one for MDM and one for no MDM, or is it only necessary if you're using MDM?

Reply:
If you do not have MDM, it's not really possible. I'll make that more obvious.